src/main/scala/mesosphere/marathon/api/akkahttp/v2/GroupsController.scala

```diff
@@ -76,4 +96,29 @@ class GroupsController(electionService: ElectionService, infoService: GroupInfoS
     }
   }
   // format: On
+
+  def authorizationSelectors(implicit identity: Identity): GroupInfoService.Selectors = {
```

Comment:
What is this method doing? Could you add docs?

Reply:
there is some weird concept of selectors in group-related methods... I will add comment but it won't make it that much clearer...